src/common/constants/constants.js

```diff
@@ -46,3 +46,7 @@ export const UNOPEN_ORDER_ABANDONED_HOLDINGS_TYPES = {
 
 export const FIELD_ARRAY_ITEM_IDENTIFIER_KEY = '__key__';
 export const OKAPI_TENANT_HEADER = 'X-Okapi-Tenant';
+
+export const HIDDEN_ORDER_LABELS_MAP = {
```

Comment:
If this constant is used in several places and should be defined here, please specify the relation with the version history. It's not entirely obvious from the name why the fields are hidden

Reply:
I believe we have to provide field names to hide, not labels

Reply:
updated